packages/fast-layouts-react/src/column/column.props.ts

```diff
@@ -31,6 +31,11 @@ export interface ColumnHandledProps extends ColumnManagedClasses {
      * The gutter size of the parent Grid component
      */
     gutter?: GridGutter;
+
+    /**
+     * The default breakpoint of the parent Grid component
```

Comment:
This *technically* isn't the breakpoint of the grid; I think we want something like:
```suggestion
     * The default breakpoint (helpful when rendering server-side).
     * Defaults to 0